src/words.js

```diff
@@ -1,6 +1,15 @@
-module
-export {
-  function words() {
-
-  }
-}
\ No newline at end of file
+"use strict";
+module.exports = {
+	words: (str) => {const result = {};
+  const arr = str.split(' ');
+  arr.forEach((word) => {
+    if (result[word]) {
+      result[word] += 1;
+    } else {
+      result[word] = 1;
+    }
+    return word;
+  });
+  return result;
+}}
+	
```

Comment:
Try to format your code well. It is nice that you have an .eslintrc file, but you did not make use of it well, so I came across a lot of spacing and formatting errors.